src/Shouldly.Tests/ShouldContain/LongStringScenario.cs

```diff
@@ -12,7 +12,7 @@ protected override void ShouldThrowAWobbly()
 
         protected override string ChuckedAWobblyErrorMessage
         {
-            get { return "longString should contain \"fff\" but does not"; }
+            get { return "longString should contain \"fff\" but was actually \"aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa\""; }
```

Comment:
Not really sure what you mean here. Can you elaborate? Maybe be open a dedicated issue and send it my way? Tx